client/Combatant/StaticCombatantViewModel.ts

```diff
@@ -17,6 +18,7 @@ module ImprovedInitiative {
             this.Initiative = combatant.Initiative();
             this.IsPlayerCharacter = combatant.IsPlayerCharacter;
             this.Tags = combatant.Tags();
+            this.AllowSuggestions = Store.Load(Store.User, "PlayerViewAllowPlayerSuggestions");
```

Comment:
Oh, good point! I think it will work fine in that case. It would still be preferable to send this setting at the encounter level.